lib/binding.js

```diff
@@ -108,8 +116,9 @@ exports.symlink = function() {
 	throw new Error('Unimplemented');
 }
 
-exports.unlink = function() {
-	throw new Error('Unimplemented');
+exports.unlink = function(path, flags, req) {
```

Comment:
this module is used by the fs module and we have to reflect the native binding interface otherwise the arguments will mismatch https://github.com/resin-io/node-lkl/blob/master/lib/fs.js#L991